spec/Observable-spec.ts

```diff
@@ -193,6 +193,80 @@ describe('Observable', () => {
       source.subscribe();
     });
 
+    it('should work when subscribe is called with no arguments and other subscription with no arguments completes', (done: MochaDone) => {
+      let nextCalled = false;
+      const source = new Observable((subscriber: Rx.Subscriber<string>) => {
+        const id = setTimeout(() => {
```

Comment:
This should be using the `rxTestScheduler` instance rather than `setTimeout` so the test can be deterministic.